src/overlays/actors/ovl_En_Wiz_Brock/z_en_wiz_brock.h

```diff
@@ -8,10 +8,14 @@ struct EnWizBrock;
 typedef void (*EnWizBrockActionFunc)(struct EnWizBrock*, GlobalContext*);
 
 typedef struct EnWizBrock {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x5C];
+    /* 0x0000 */ DynaPolyActor dyna;
+    /* 0x015C */ char unk_15C[0x44];
     /* 0x01A0 */ EnWizBrockActionFunc actionFunc;
-    /* 0x01A4 */ char unk_1A4[0xC];
+    /* 0x1A4 */ s16 despawnCounter; // Counter for despawing blocks (Max of 0x25)
+    /* 0x1A6 */ s16 unk1A6;
```

Comment:
```suggestion
    /* 0x1A6 */ s16 unk1A6; // set but not used
```

Reply:
I wasn't sure to put this or not, it could be used in the actual wizrobe actor, I didn't check though

Reply:
Oh true. Could leave a TODO instead if you want. Also, unk_1A6.